easybuild/easyconfigs/t/TranscriptClean/TranscriptClean-2.0.2-foss-2020b.eb

```diff
@@ -0,0 +1,35 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Denis Kristak (INUITS)
+
+easyblock = "Tarball"
+
+name = 'TranscriptClean'
+version = '2.0.2'
+
+homepage = 'https://github.com/mortazavilab/TranscriptClean'
+description = """TranscriptClean is a Python program that corrects mismatches, microindels,
+and noncanonical splice junctions in long reads that have been mapped to the genome."""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+source_urls = ['https://github.com/mortazavilab/TranscriptClean/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['27326c10d04a2e2fef29865fc3f040a8db705ea4a4aeb6cf48a36c102f00381c']
+
+dependencies = [
+    ('R', '4.0.4'),
```

Comment:
You'll need to stick to R 4.0.3 to avoid that the tests yell at you, unless R 4.0.4 is really needed, then you should make sure`-R-%(rver)s` is added to `versionsuffix`